source/tutorials.txt

```diff
@@ -8,14 +8,16 @@ Tutorials
    :titlesonly:
    :hidden:
    
+   Template App Tutorials <https://www.mongodb.com/docs/realm/tutorial/>
+   Create a Template App </tutorial/create-template-app>
    GitHub Issue Tracker </tutorial/backend>
    Set up JWT Authentication </tutorial/jwt>
 
-Learn how to:
 
-- :doc:`Write a GitHub Issue Tracker with Atlas App Services </tutorial/backend>` 
-- :doc:`Set up JWT Authentication with App Services </tutorial/jwt>`
 
-To explore using GraphQL with the Web SDK, see:
+Learn how to use Atlas App Services' features and integrate external tools. 
+
+- :doc:`Write a GitHub Issue Tracker with Atlas App Services </tutorial/backend>`
+- :doc:`Set up JWT Authentication with App Services </tutorial/jwt>`
+- `Template App Tutorials <https://www.mongodb.com/docs/realm/tutorial/>`_
```

Comment:
please replace this link with a `:ref:` to the realm-docs repo. Refs between the docs-realm and docs-app-services repos are shared